### DISCUSSION TITLE
Should SC remove bicycle=designated for sharrows and advisory lanes?

### DISCUSSION THREAD

**Discussion by SLMapper** [streetcomplete/StreetComplete#4730] [Q&A]
I think `cycleway:*:bicycle=designated` is not correct for sharrows (also not for advisory lanes) as currently present here: https://www.openstreetmap.org/way/30073084
(tag was manually added in https://www.openstreetmap.org/changeset/91233650)

Should SC remove this when changing type to sharrow or advisory lane?

Currently existing minimum [218 times](https://overpass-turbo.eu/s/1pJE) in Germany.

**Comment by matkoniecz** [comment]
It is not very useful, but sharrows are marking road as designated for cyclists.

**Reply by SLMapper** [reply]
In my understanding `designated` means there is a sign, which is not the case here

**Reply by matkoniecz** [reply]
sharrows are lane markings which are a type of sign (at least in Poland there are "vertical traffic signs" and "horizontal traffic signs")

**Comment by BalooUriza** [comment]
No.  How much more designated are you expecting than literally painted on
the lane?
